RocketChat/Rocket.Chat
# Chore: Disable Sessions Aggregates tests locally

## Proposed changes (including videos or screenshots)


Disable Session aggregates tests in local environments
For context, refer to: #20161 


## Issue(s)

Locally, `sessions` tests where failing due to an external problem with MongoDB. They are working as expected on CI. The goal of this MR is to disable the test when running locally but allowing it to run on CI.

## Steps to test or reproduce

1. Run `npm run testunit` on local environment (linux or WSL)
2. Session aggregates test should fail

## Further comments
